File: Queue/QueueJava/Manejador.java
package Queue.QueueJava;

public class Manejador {
    
    public static void main(String[] args ) {

        Queue queue = new Queue();
        queue.enqueue('J');
        queue.enqueue('A');
        queue.enqueue('S');
        queue.enqueue('O');
        queue.enqueue('N');
        queue.printQueue();
        
        queue.dequeue();
        System.out.println("\n");

        queue.printQueue();
        
        

    }
    
}

File: Queue/QueuePython/Manejador.py
from Queue import Queue
def __main__():

    queue = Queue()
    queue.enqueue(1)
    queue.enqueue(2)
    queue.enqueue(3)
    queue.enqueue(4)
    queue.enqueue(5)
    queue.enqueue(6)
    queue.print_queue()
    queue.dequeue()
    queue.print_queue()
    
if(__name__ == '__main__'):
    __main__()
